db-commons/src/main/resources/reference.conf

```diff
@@ -57,6 +57,20 @@ bitcoin-s {
     port = 9050
   }
 
+  tor {
+    # You can enable Tor for incoming connections
+    enabled = false
+    control = "127.0.0.1:9051"
```

Comment:
I think we might want to have the same `host:port` format for proxy as well

Reply:
:+1: